Common/DtaCommand.cpp

```diff
@@ -204,7 +204,7 @@ DtaCommand::complete(uint8_t EOD)
     hdr->pkt.length = SWAP32((bufferpos - sizeof (OPALComPacket))
                              - sizeof (OPALPacket));
     hdr->cp.length = SWAP32(bufferpos - sizeof (OPALComPacket));
-	if (bufferpos > 2048) {
+	if (bufferpos > 4196) {
```

Comment:
Was this intended to be `4096`?  If not, is there anything special about `4196` ?